conf/application.conf

```diff
@@ -22,7 +22,7 @@ application.global="femr.util.startup.Global"
 
 #Register MySQL database settings
 db.default.driver="com.mysql.jdbc.Driver"
-db.default.url="jdbc:mysql://localhost/femr?characterEncoding=UTF-8"
+db.default.url="jdbc:mysql://localhost/femr2?characterEncoding=UTF-8&useLegacyDatetimeCode=false&serverTimezone=UTC"
```

Comment:
make sure the connection string in application.conf and application.example.conf match the default database name of femr

`jdbc:mysql://localhost/femr?characterEncoding=UTF-8&useLegacyDatetimeCode=false&serverTimezone=UTC`